pkg/apis/triggers/v1alpha1/event_listener_defaults.go

```diff
@@ -31,5 +31,37 @@ func (el *EventListener) SetDefaults(ctx context.Context) {
 			triggerSpecBindingArray(el.Spec.Triggers[i].Bindings).
 				defaultBindings()
 		}
+		// Remove Deprecated Resource Fields
+		el.Spec.updatePodTemplate()
+		el.Spec.updateServiceType()
+	}
+}
+
+// To be Removed in a later release
+func (spec *EventListenerSpec) updatePodTemplate() {
```

Comment:
could we add a couple of test cases for these in https://github.com/tektoncd/triggers/blob/master/pkg/apis/triggers/v1alpha1/event_listener_defaults_test.go#L28